packages/shared/lib/utils/analytics.ts

```diff
@@ -19,11 +19,6 @@ export enum AnalyticsTypes {
     CONNECTION_INSERTED = 'server:connection_inserted',
     CONNECTION_LIST_FETCHED = 'server:connection_list_fetched',
     CONNECTION_UPDATED = 'server:connection_updated',
-    ONBOARDING_0 = 'onboarding:step_1_authorize',
```

Comment:
I think there was a reason we did server side tracking instead of client side, but I don't remember why now, @bastienbeurier do you remember -- if any?

Reply:
yeah good point indeed. I don't we have used it on Posthog but tbd

Reply:
Server-side tracking is more reliable because it can't be suppressed by ad blockers. Typically, our Interactive Demo funnel would be more accurate if tracked server-side. 

But client-side is important to surface the right session recordings.

Reply:
I'm not sure it's worth fighting for but okay, I'll put that back 👌🏻